atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/PrimaryKeyConstraintNames.java

```diff
@@ -15,21 +15,15 @@
  */
 package com.palantir.atlasdb.keyvalue.dbkvs.impl.oracle;
 
-import org.apache.commons.lang3.StringUtils;
+import com.palantir.atlasdb.AtlasDbConstants;
 
-import com.palantir.atlasdb.keyvalue.dbkvs.OracleTableNameMapper;
+public final class PrimaryKeyConstraintNames {
```

Comment:
The `get()` method seems Oracle-specific - I would think a call to `PrimaryKeyConstraintNames.get(foo)`, which adds the Oracle prefix, should have Oracle somewhere in the name? I know you plan to extend this to Postgres in some way, so feel free to push back on this (or maybe this is some intermediate step along the way)